### PR TITLE
Fix author's XML node name in `wp export`

### PR DESCRIPTION
**Pull request by rodrigoprimo**
See https://core.trac.wordpress.org/ticket/22435#comment:49

### CODE REVIEW

**Comment by danielbachhuber** [comment]
Why weren't imports broken if this has been broken forever?

**Comment by mavin** [comment]
+1 I was just going to make a PR for this same issue however I can't explain why the importer isn't failing.

I suspect it's because of this:
https://plugins.trac.wordpress.org/browser/wordpress-importer/trunk/wordpress-importer.php#L235

**Comment by rodrigoprimo** [comment]
Only the user name was used by the importer. It was getting this information from the posts as @mavin said. User e-mail, display name and other info were missing. I have added a test case that will illustrate the problem.

File: features/export.feature [hunk 1]
```diff
@@ -201,3 +201,29 @@ Feature: Export content.
...
+    Then STDOUT should not be empty
+
+    When I run `wp import {EXPORT_FILE} --authors=create`
+    Then STDOUT should not be empty
+
+    When I run `wp user get user --field=display_name`
+    Then STDOUT should be:
+      """
+      Test User
+      """
```
Comment (danielbachhuber) [inline features/export.feature]: Can you make sure there's a newline here?

**Comment by danielbachhuber** [comment]
> I have added a test case that will illustrate the problem.

Thanks!

**Comment by rodrigoprimo** [comment]
Done, I keep forgetting about this.

**Pull request merged**